app/dashboard/templates/dashboard/sponsors.html

```diff
@@ -266,6 +266,7 @@
       document.hackathon_id = "{{ hackathon.id | safe }}";
       document.hackathonObj = JSON.parse(document.getElementById('hackathon-object').textContent);
       document.activePanel = parseInt({{panel | safe}});
+      document.is_sponsor = {{ is_sponsor|yesno:'true,false' }}
```

Comment:
@octavioamu side note -> do wanna attach this to the DOM  ?

Reply:
I think yes because then is handle with vuejs to show/hide the dashboard link

Reply:
Removed, thanks!

Reply:
ohh I see I got confused with isSponsor